pasqal_cloud/client.py

```diff
@@ -218,8 +229,8 @@ def _get_workload(self, workload_id: str) -> Dict[str, Any]:
     def _cancel_workload(self, workload_id: str) -> Dict[str, Any]:
         workload: Dict[str, Any] = self._request(
```

Comment:
```suggestion
        workload: JSendPayload= self._request(
```

Reply:
I think this was a change I forgot to revert  which is why it's alone

Reply:
cool, makes sense.
Figured that once I saw it I'd save you another few moments living in Python land, since it was what broke your mypy ;)